src/core/Private/esettings.cpp

```diff
@@ -268,6 +270,51 @@ eSettings::eSettings(const int cpuThreads,
                      fTimelineHighlightRowColor,
                      "timelineHighlightRowColor",
                      QColor(255, 0, 0, 15));*/
+    // Grid default colors, TODO: move them to ThemeSupport
```

Comment:
This is for config items in the preferences dialog, since snap/grid uses it own settings dialog these items should not be here.

Reply:
You are right, where could we place them?